ase/scripts/create_ase_simbuild_env.sh

```diff
@@ -27,6 +27,12 @@
 
 set -e
 
+##
```

Comment:
Should this script be deprecated? If so, should we print out a deprecation notice when this script is run?

Reply:
Probably. I already asked Dipti and Rahul about it.